test/unit/coffee/WebsocketLoadBalancerTests.coffee

```diff
@@ -17,6 +17,7 @@ describe "WebsocketLoadBalancer", ->
 			"./HealthCheckManager": {check: sinon.stub()}
 			"./RoomManager" : @RoomManager = {eventSource: sinon.stub().returns @RoomEvents}
 			"./ChannelManager": @ChannelManager = {publish: sinon.stub()}
+			"./ConnectedUsersManager": @ConnectedUsersManager = {refreshClient: sinon.stub()}
```

Comment:
What's this change about?

Reply:
The tests worked if you had redis running (which apparently we do while running the unit tests on Jenkins) so this needed to be stubbed out.

Reply:
ah nice, thanks